### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by Heraclion** [raknjak/CIMembership#10]
I want to give you a language script that you are store the language in dbase,and delete,modify,using api keys to autotranslate or manually translate to implement in this script

**Comment by raknjak** [comment]
Yeah this is a good one. I probably should make a branch for this, I'm so low on time though.

**Comment by raknjak** [comment]
Decision: store all language entries in the database and find a good way to preload only the ones needed per request.
